config/config.json

```diff
@@ -0,0 +1,13 @@
+{
+    "database" : {
+        "port": "27017",
+        "uri": "localhost",
+        "collection": "PokeData"
+    },
+    "twitter" : {
+        "consumer_key": "46yqEVwOOkf7dtN5Zfqxw4Nzu",
```

Comment:
No secrets here!

I urge you to revoke your twitter keys and create a placeholder `config.json`. These files can be read by **EVERYONE** on the web! In the worst case, malicious coders can use your keys to do nasty things on twitter, and you will be responsible for it!